### PR TITLE
Added logo and a link

### PR DESCRIPTION
**Pull request by BraydenGirard**

### CODE REVIEW

**Comment by swyxio** [comment]
Deploy Preview is done!

Built with commit bbc7947c0ac602d239a5c4b9edefab20a2ae8345

https://deploy-preview-379--svelte-community.netlify.app

File: whos-using-svelte/organisations/daydream.svg [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
...3LnuQIAkQYGwHJhpXgi1+hzBOxD4QpC8UffWVl3cN2gu5vLzkbYCZIIwdASh7gukQBorqUICk+VbOnX1tV6AdyKWlPT7KAIkbCQVCAUUgBiACUlqFQ3cLbGpS4lrprO2BAJw/d3YqaAG5tLTHk4QAIARIQLlSBs1QhRCRBC0SwSWHDyuGd0mAAEAhLlw4PxE0QRpgAombS1xUoooiBiKgMCrMFs7mZkBljXCZmgSxsfH6WFCpAU0FZG0DpAhUte3LajqIRDibQwGJ0C+5otV+ADY33+gFlRrQ/E+YQURAM2sYY/ssYNsxRkWFkgLMfpfMnYU8SVy8uNEBldwMtjBJiJZKqQhUFAJCGg19qhDV5BYiEuYln5VM3dYlCnOzMj2J0WizABVJgFr4WGs+CSCqAUADqIqWJteyTbEM0FqoBqzhcnW3t0YJVP7+518KtZ0ggIpCK58ztVRNwQysflIgotFNmFwgqFJHfz8ogASaetfWVj2lVUijgioCaZpgbm3QWN89ikYbiAoaaaBNgFdViGg4TL5GpnwyvfQEESrgmBIBAHfv/sVeQFVoVEZtrJF0mEaiqqJJQbXUlVJYlYqkq2QvoJk7f7lz50+mQFGFaBNgokKqAdBUscBRM72aSqGGg7BQsL2N+oGK99juXIu3b/9B1UpRlQhoplRok/ulpujX5J/m51nASKtkDlSD5oCj0RZ7PzB+/e13tgra5lo+U7UU1R6szaWVD3ZunzKia9jRaIu9Slr55dYtJsBGU1CEdyngVO1qZOaH7ARJnt5qX6zNbIATIQHg5s2fqVl6MUAxlU01aaFrBfM82X7/lgrWJQecCgkAP/z4E7XRkGbM/4r0wiynagfQVLSbxa7EcdE82truUO/qH7Hvvv0+KJoiObttUrubBy3aJ94uPdE8FyQAfPnV1y0os6jPTF5+DUnwtR4fBHoCpfLDuSAB4Pr1L9gHaJHdieIJgHUZBzgzJAB8+tnnKZjGAY678nLAaX64ECQAXLv2CWUGQCvzAM4NCQBXr37MWqlxgLNE8qCQAPDBhx+xCJQpn1/zAC4MCQDvvvc+pylYR/IsgINAAsCVt99hx/8WNHFeBoEEgMuXrzDBDWDivAwGCQCXLr3FoUycl0EhAWB7+1JS1MoigMD/AAkAW9tv5vZeeL3/AOJENMzt51iTAAAAAElFTkSuQmCC"/>
```
Comment (dreitzner) [inline whos-using-svelte/organisations/daydream.svg]: is there a way to not use PNGs inside of your svg?

**Review by dreitzner — CHANGES_REQUESTED** [review]
- [ ] fix png inside of svg

File: whos-using-svelte/organisations/daydream.svg (hunk 1 above)
Comment (BraydenGirard) [reply whos-using-svelte/organisations/daydream.svg]: I have tried to remove the png but I'm not sure if the source eps I am working from has it in it. Is this latest commit any better?  Sorry for confusion.

**Review by dreitzner — CHANGES_REQUESTED** [review]
please pull in the latest changes from master and add your info to https://github.com/sveltejs/community/blob/master/whos-using-svelte/WhosUsingSvelte.js

**Comment by BraydenGirard** [comment]
Done!

**Review by dreitzner — COMMENTED** [review]
Looks good 👍

**Review by dreitzner — APPROVED** [review]

**Pull request merged**